[2022-02-10 01:27:02,083] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: homework_week3.gcs_to_gcs_task manual__2022-02-10T01:26:58.473213+00:00 [queued]>
[2022-02-10 01:27:02,127] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: homework_week3.gcs_to_gcs_task manual__2022-02-10T01:26:58.473213+00:00 [queued]>
[2022-02-10 01:27:02,128] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 01:27:02,128] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-10 01:27:02,128] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 01:27:02,172] {taskinstance.py:1259} INFO - Executing <Task(GCSToGCSOperator): gcs_to_gcs_task> on 2022-02-10 01:26:58.473213+00:00
[2022-02-10 01:27:02,184] {standard_task_runner.py:52} INFO - Started process 8077 to run task
[2022-02-10 01:27:02,186] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'homework_week3', 'gcs_to_gcs_task', 'manual__2022-02-10T01:26:58.473213+00:00', '--job-id', '502', '--raw', '--subdir', 'DAGS_FOLDER/homework_week3_dag.py', '--cfg-path', '/tmp/tmp_a_siday', '--error-file', '/tmp/tmps3ak67u2']
[2022-02-10 01:27:02,188] {standard_task_runner.py:77} INFO - Job 502: Subtask gcs_to_gcs_task
[2022-02-10 01:27:02,404] {logging_mixin.py:109} INFO - Running <TaskInstance: homework_week3.gcs_to_gcs_task manual__2022-02-10T01:26:58.473213+00:00 [running]> on host 0fd4ea805f5e
[2022-02-10 01:27:02,627] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=homework_week3
AIRFLOW_CTX_TASK_ID=gcs_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-10T01:26:58.473213+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-10T01:26:58.473213+00:00
[2022-02-10 01:27:02,630] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-10 01:27:02,631] {gcs_to_gcs.py:346} INFO - Delimiter ignored because wildcard is in prefix
[2022-02-10 01:27:03,068] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=homework_week3, task_id=gcs_to_gcs_task, execution_date=20220210T012658, start_date=20220210T012702, end_date=20220210T012703
[2022-02-10 01:27:03,139] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 01:27:03,254] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
